infra/scripts/cometbft.toml

```diff
@@ -34,7 +35,7 @@ dependencies = ["cometbft-pull", "docker-network-create"]
 script = """
 id=`docker exec ${CMT_CONTAINER_NAME} cometbft show-node-id`
```

Comment:
This would be a good spot for grabbing the host IP, although I am not exactly sure if that can be used inside docker.